client/src/components/tags/TagsList.js

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from "react";
+import { getTags } from "../../managers/tagManager";
+import { Button, Table } from "reactstrap";
+
+export default function TagsList() {
+    const [tags, setTags] = useState([])
+
+    useEffect(() => {
+        getTags().then(setTags)
+    }, [])
+
+    return (
+        <>
+        <Table>
+            <thead>
+                <tr>
+                    <th>Name</th>
+                </tr>
+            </thead>
+            <tbody>
+                {tags.map((t) => (
+                    <tr key={t.id}>
+                        <th scope="row">{`${t.name}`}</th>
+                        <Button>Add New Tag</Button>
```

Comment:
Maybe move the "add new tag" button outside of the map function.

Reply:
Agreed/done